resources/linux/debian/postinst.template

```diff
@@ -35,9 +35,18 @@ if [ "@@NAME@@" != "code-oss" ]; then
 	eval $(apt-config shell APT_TRUSTED_PARTS Dir::Etc::trustedparts/d)
 	CODE_TRUSTED_PART=${APT_TRUSTED_PARTS}microsoft.gpg
 
+	. /usr/share/debconf/confmodule
+	db_get @@NAME@@/add-microsoft-repo || true
+
 	# Install repository source list
 	WRITE_SOURCE=0
-	if [ ! -f $CODE_SOURCE_PART ] && [ ! -f /etc/rpi-issue ]; then
+	if apt-cache policy | grep -Eq "https:\/\/packages\.microsoft\.com\/repos\/code"; then
```

Comment:
Would `grep -q https://packages.microsoft.com` work instead?
When I run `apt-cache policy | grep https://packages.microsoft.com` I get two results that both contain `https://packages.microsoft.com/ubuntu/22.04/prod` rather than /repos/code.

Reply:
If the package is included in either, that's fine with me. Will change it

Reply:
Just realized that there is one issue with this: It would also match `/repos/vscode`, which would be the old repo that requires a migration. On the other hand, that migration isn't really clean, as `/repos/vscode` could potentially still be in the apt sources it it isn't inside `$CODE_SOURCE_PART`

Reply:
Never mind... It only checks in $CODE_SOURCE_PART, so the migration is fine. I'll just reorder the ifs.

Reply:
Should be resolved now, only thing I don't know (which you could probably verify) is whether all repos on `packages.microsoft.com` contain the `code` package